week2/homework/src/index.js

```diff
@@ -1,3 +1,95 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+let command = process.argv[2];
+let value = process.argv[3];
+let listFile = fs.readFileSync('./data.json', 'utf8');
+let listArr = JSON.parse(listFile);
+
+switch (command) {
+  case 'list':
+    viewTasks();
+    break;
+  case 'add':
+    addTask();
+    break;
+  case 'help':
+  case undefined:
+    let helpFile = fs.readFileSync('./help.txt', 'utf8');
+    console.log(helpFile);
+    break;
+  case 'remove':
+    removeTask(value);
+    break;
+  case 'reset':
+    clearTasks();
+    break;
+
+  case 'update':
+    updateTasks(value);
+    break;
+
+  default:
+    console.log('wrong command check the help menu by typing help');
+}
+
+function viewTasks() {
+  if (!listArr || listArr.length == 0) {
+    console.log('no toDos sorted');
+  } else {
+    listArr.forEach(element => {
+      console.log(element);
+    });
+  }
+}
+
+function addTask() {
+  if (process.argv.length < 4) {
+    console.log('check the help menu by typing help');
+  } else {
+    let taskLine = '';
+    for (let i = 3; i <= process.argv.length - 1; i++) {
```

Comment:
you don't need to do this, but it's a clever trick!
when adding a todo, you should call it with `node index.js add "my todo"`. this way you don't have to check for all the `argv` after the command, you can just get that one `argv`.